eos-update-flatpak-repos

```diff
@@ -1742,15 +1745,18 @@ def _migrate_installed_flatpaks():
                                                     arch, new_branch)
                         refs.append(new_ref)
 
-                    if migrate_extra:
-                        migrate_extra_data(ref, new_ref)
+                    migrate_extra_data(ref, new_ref)
 
-                    logging.info('Uninstalling old ref: {}'.format(old_ostree_ref))
-                    inst.uninstall_full(Flatpak.UninstallFlags.NO_PRUNE |
-                                        Flatpak.UninstallFlags.NO_TRIGGERS,
-                                        kind, name, arch, branch)
-
-                    triggers_needed = True
+                    try:
+                        logging.info('Uninstalling old ref: {}'.format(old_ostree_ref))
+                        inst.uninstall_full(Flatpak.UninstallFlags.NO_PRUNE |
+                                            Flatpak.UninstallFlags.NO_TRIGGERS,
+                                            kind, name, arch, branch)
+
+                        triggers_needed = True
```

Comment:
I think this needs to be set unconditionally since if we have reached this point, we have definitely installed the new ref, so triggers (update exported files etc.) do need to run.

Reply:
Good point